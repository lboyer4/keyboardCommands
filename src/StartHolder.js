import React, { Component } from 'react';
import './StartPage.scss';

class StartPage extends Component {
  constructor() {
    super()

  }
  render() {
    return (
    	<section>
    		<

    	</section

  }
}

export default StartPage;