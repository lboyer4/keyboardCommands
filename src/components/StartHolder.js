import React, { Component } from 'react';
import '../SCSS/_StartHolder.scss';
import Start from './Start.js';

class StartHolder extends Component {

  
  render() {
    return (
    	<section className="top-of-page"> 
    		<header>
    			<h1 className="title">Key Commands</h1>
    		</header>
    			<Start setLevel={this.props.setLevel} />
    	</section>
    )
  }
}

export default StartHolder;