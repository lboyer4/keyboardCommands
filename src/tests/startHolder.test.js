import React from 'react';
import StartHolder from '../components/StartHolder';
import { shallow } from 'enzyme'; 


describe('StartHolder', () => {
  
	const mockSetLevel = jest.fn();
	const mockSetWrongCards = jest.fn();
	const mockIncorrectCards = [{test: 'test one'
	}, {test: 'test two'}]

  it("should match the snapshot with all data passed in", () => {
   	let wrapper = shallow(
   		<StartHolder 
	      setLevel={mockSetLevel}
	      setWrongCards={mockSetWrongCards}
	      incorrectCards={mockIncorrectCards}
       />)
   	expect(wrapper).toMatchSnapshot();
  });
});