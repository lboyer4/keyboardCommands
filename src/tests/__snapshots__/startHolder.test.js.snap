// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartHolder should match the snapshot with all data passed in 1`] = `
<section
  className="start-container"
>
  <header>
    <h1
      className="title"
    >
      Compelling Key Commands
    </h1>
    <p
      className="description"
    >
      Help a unicorn fly over the rainbow by practicing key Key Commands
    </p>
  </header>
  <Start
    incorrectCards={
      Array [
        Object {
          "test": "test one",
        },
        Object {
          "test": "test two",
        },
      ]
    }
    setLevel={[MockFunction]}
    setWrongCards={[MockFunction]}
  />
</section>
`;
